csv2wiki/__init__.py

```diff
@@ -924,7 +924,11 @@ def __init__(self, csv_file, config):
         self.row_count           = None  # will be num rows not counting header
         
         self._config = config or {}
-        self._csv_fh = open(csv_file)
+        try:
+            self._csv_fh = open(csv_file)
+        except TypeError:
+            # EAFP for when a stream is coming in rather than a filename
+            self._csv_fh = csv_file
```

Comment:
Would be clearer to rename 'csv_file' to say 'csv_file_or_stream', or 'csv_input', or just 'csv', or similar.  

Also seems like there should be a better way than try/except here.  Maybe using `type()`([like this](https://stackoverflow.com/a/2225066))?

Reply:
Can't do csv because of the package, but will use csv_input :).  As for doing try/except, that's a python idiom.  It's very pythonic to use exceptions for control flow (see https://en.wikipedia.org/wiki/Python_syntax_and_semantics#Exceptions)

Reply:
Ah, I see.  Makes sense now.  Thanks for the link!